README.md

```diff
@@ -1,55 +1,61 @@
-* `NEW_ACCOUNT_AMOUNT` -- default `10000000000`, integer;
-* `ACCOUNT_CREATOR_KEY` -- JSON key of account used to create accounts
-* `ACCOUNT_RECOVERY_KEY` -- JSON key of  account used to recover accounts
-* `TWILIO_FROM_PHONE` – phone number from which to send SMS with security code (international format, starting with `+`)
-* `TWILIO_ACCOUNT_SID` – account SID from Twilio (used to send security code)
-* `TWILIO_AUTH_TOKEN` – auth token from Twilio (used to send security code)
+Microservice used to create NEAR accounts
 
 ## Local Development
 
 ### Requirements
 
-1) Install latest Node.js LTS release.
-2) Install [HTTPie](http://httpie.org/).
+1) Install latest Node.js LTS release
+2) Install [HTTPie](http://httpie.org/)
 
 ### Install dependencies
 
-```
-npm install
-```
+    yarn
 
 ### Create database
 
-Create `accounts_development` Postgres DB with `fiddle` user/password.
+Make sure you have PostgreSQL installed and running. On macOS, you can use [Postgres.app](https://postgresapp.com/).
+
+Create `accounts_development` Postgres DB with `helper` user/password. You can do this from within `psql` using:
+
+    create user helper with superuser password 'helper';
+    create database accounts_development with owner=helper;
+
 After that:
 
-```
-npm run migrate
-```
+    yarn migrate
 
 You can also modify DB config in `config/config.js` to use different connection settings, etc.
 
-### Run server
+### Environment Variables
+
+Create a `.env` file, copy in the default values from `.env.sample`. Read this file for information about how to change configuration settings to suit your needs.
+
+By default, it assumes that you're running a local node and local network. To do this:
+
+* clone [nearcore](https://github.com/nearprotocol/nearcore)
+* in your nearcore directory, get a local network running (at the time of writing, the command was `./scripts/start_localnet.py`)
+
+Note that you need to add a `ACCOUNT_CREATOR_KEY` and `ACCOUNT_RECOVERY_KEY` to your `.env`. Running `nearcore` locally created a `~/.near/validator_keys.json` file for you. Copy the contents of this file and paste them as a single line for each of the `*_KEY` values in your `.env`.
 
-Make sure that before running service there are appropriate env variables set (can be put in  `.env` file in root directory of project):
+### Run server
 
-```
-npm start
-```
+    yarn start
 
 ### Create account
-```
-http post http://localhost:3000/account newAccountId=nosuchuseryet.near newAccountPublicKey=22skMptHjFWNyuEWY22ftn2AbLPSYpmYwGJRGwpNHbTV
-```
+
+    http post http://localhost:3000/account newAccountId=nosuchuseryet newAccountPublicKey=22skMptHjFWNyuEWY22ftn2AbLPSYpmYwGJRGwpNHbTV
+
+
+## Running tests
+
+### Create database
+
+Follow the instructions above for creating the development database and `helper` user. Then create an `accounts_test` database using `psql`:
+
+    create database accounts_test with owner=helper;
+
+### TODO
+
+How to get the tests running?
```

Comment:
This is a question for you, @vgrichina

Reply:
got tests running on CI here https://github.com/nearprotocol/near-contract-helper/pull/89
same setup should work locally